packages/terra-application-header/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "terra-application-links",
```

Comment:
This file is located at `packages/terra-application-header/package.json`, though it looks like its using `terra-application-links` in places it should be using `terra-application-header`.

The name key of this package should be updated to `terra-application-header`.